documenting.rst

```diff
@@ -1578,14 +1578,247 @@ Sphinx_, blurb_ and python-docs-theme_ packages from PyPI)::
+
+
+Starting a new translation
+--------------------------
+
+First subscribe to the `doc-sig
+<https://mail.python.org/mailman/listinfo/doc-sig>`_ mailing list,
+introduce yourself and the translation you're starting.
+
+Then you can bootstrap your new translation by using our `cookiecutter
+<https://github.com/JulienPalard/python-docs-cookiecutter>`__.
+
+The important steps looks like this:
+
+- Create the github repo (anywhere), with the right hierarchy (using the
+  cookiecutter).
+- Gather people to help you translating, you can't do it alone.
+- You can use any tool to translate, as long as you can synchronize with git,
+  some are using Transifex, some are using only github, you can choose another
+  way, it's up to you.
+- Ensure we updated this page to reflect your work and progress, either via a
+  PR, or by asking on the doc-sig mailing list.
+- When ``tutorial/``, ``library/stdtypes`` and ``library/functions``
+  are complete, ask on doc-sig for your language to be added in the
+  language picker on docs.python.org.
+
+
+PEP 545: TL;DR
+--------------
+
+The :PEP:`545` may be long to read, so here are the interesting points:
+
+- Each translation is assigned an appropriate lowercased language tag,
+  with an optional region subtag (if needed), glued with a dash, like
+  ``pt-br`` or ``fr``.
+
+- Each translation is under CC0 and marked as so in the README (as in
+  the cookiecutter).
+
+- Translations files are hosted on
+  ``https://github.com/python/python-docs-{LANGUAGE_TAG}`` (not
+  mandatory to start a translation, but mandatory to land on
+  ``docs.python.org``).
+
+- Translations having completed ``tutorial/``, ``library/stdtypes``
+  and ``library/functions`` are hosted on
+  ``https://docs.python.org/{LANGUAGE_TAG}/{VERSION_TAG}/``.
+
+
+How to get help
+---------------
+
+Discussions about translations occur on the `doc-sig
+<https://mail.python.org/mailman/listinfo/doc-sig>`_ mailing list,
+and there's a freenode IRC channel, ``#python-doc``.
+
+
+Translation FAQ
+---------------
+
+Which version of Python documentation should be translated?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Concensus is to work on current stable, you can then propagate your
+translation from a branch to another using `pomerge
+<https://pypi.org/p/pomerge>`__.
+
+
+Are there some tools to help in managing the repo?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here's what's we're using:
+
+- `pomerge <https://pypi.org/p/pomerge>`__ to propagate translation
+  from a files to others.
+- `pospell <https://pypi.org/p/pospell>`__ to check for typo in po files.
+- `powrap <https://pypi.org/p/powrap>`__ to rewrap the ``.po`` files
+  before committing, this helps keeping git diffs short.
+- `potodo <https://pypi.org/p/potodo>`__ to list what needs to be translated.
+
+
+
+How a coordinator is elected?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no election, each country have to short this out, here's some
```

Comment:
```suggestion
There is no election, each country have to sort this out.  Here are some suggestions.
```
spelling

Reply:
```suggestion
There is no election, each country have to sort this out. Here are some suggestions
```
Maybe no need for two spaces after the `.` ? 😉